src/screens/PasswordChanged.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet } from 'react-native';
+import { useFonts } from 'expo-font';
+import Button from '../components/ButtonComponent';
+import successSticker from '../../assets/successSticker.png';
+import strings from '../Strings';
+
+const PasswordChanged = ({ navigation }: { navigation: any }) => {
+  useFonts({
+    'Roboto-Bold': require('../../assets/fonts/Roboto/Roboto-Bold.ttf'),
+    'Roboto-Regular': require('../../assets/fonts/Roboto/Roboto-Regular.ttf'),
+  });
+
+  function handleBackToLogin() {
+    navigation.navigate('LoginScreen');
+  }
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.body}>
+        <Image source={successSticker} testID="success-sticker" />
+        <Text style={styles.title}>{strings.passwordChanged.title}</Text>
+        <Text style={styles.subtitle}>{strings.passwordChanged.subtitle}</Text>
+        <Button
+          text={strings.passwordChanged.button}
+          handleClick={handleBackToLogin}
+          testID="back-to-login-button"
+        />
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
It would be great to move styling to a central place so other UI can use it, but we can think about that for future changes :)